ui/components/component-library/button-primary/button-primary.js

```diff
@@ -28,7 +28,7 @@ ButtonPrimary.propTypes = {
    */
   className: PropTypes.string,
   /**
-   * Boolean to change button type to Danger when true
+   * When true, ButtonSecondary color becomes Danger.
```

Comment:
Should be ButtonPrimary?
```suggestion
   * When true, `ButtonPrimary` color becomes Danger.
```